public/app/features/variables/adhoc/picker/AdHocFilter.tsx

```diff
@@ -12,6 +12,7 @@ interface Props {
   addFilter: (filter: AdHocVariableFilter) => void;
   removeFilter: (index: number) => void;
   changeFilter: (index: number, newFilter: AdHocVariableFilter) => void;
+  getTagKeysOptions?: any;
```

Comment:
Let's add a doc comment for this. This particular prop is hard to figure out unless you go pretty deep into the implementation details.